sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/Internals/AzureMonitorTransmitter.cs

```diff
@@ -37,7 +37,18 @@ public AzureMonitorTransmitter(AzureMonitorExporterOptions options)
 
             options.Retry.MaxRetries = 0;
             ConnectionStringParser.GetValues(options.ConnectionString, out _instrumentationKey, out string ingestionEndpoint);
-            _applicationInsightsRestClient = new ApplicationInsightsRestClient(new ClientDiagnostics(options), HttpPipelineBuilder.Build(options), host: ingestionEndpoint);
+
+            HttpPipeline pipeline = null;
+            if (credential != null)
```

Comment:
do we need some informational log here to show credential was provided?

Reply:
Yes. It will help in the troubleshooting process. Will get the code updated.